facebook/delphiFacebook/R/contingency_run.R

```diff
@@ -286,7 +8,16 @@ set_aggs <- function() {
 #'
 #' @export
 run_contingency_tables <- function(params) {
-  aggs <- set_aggs()
+  if ( !is.null(params$aggs_in) && file.exists(params$aggs_in) ) {
```

Comment:
it probably would save some confusion if you issued a warning if the parameter is not null but the file does not exist